### ISSUE TITLE
Two corkboards - character arc and (the current) story presentation arc

### ISSUE THREAD

**Issue by david-wendelken** [story-apps/starc#109]
Characters have an arc to their development.  Things happen to them in an order.

Stories have a presentation arc.  Things are revealed to the audience in an order.

The two orders are often, but not always, the same.  A flashback scene is an example of when a story presentation order is different from a character arc.

It would be great to be able to see both versions on the corkboard.

**Comment by dimkanovikov** [comment]
Yes! We plan to add such a module. It will be not a corkboard, but a timeline with scenes and additional events.

**Comment by hyuri** [comment]
Could sync with the Outline > **Arcs** suggestion here: #95

**Comment by dimkanovikov** [comment]
Absolutely, timeline will be just an another "view" for the same screenplay document, which allow you manage scenes in the chronological order just inside it, plus show all other information about them and vise versa. All them will be connected inside a huge script data model.

**Comment by imgforge** [comment]
Working with several episodes and multiple storylines in a non-linear format is a challenge -- I don't know of any screenwriting software that can address this -- I use Aeon Timeline extensively, but to somehow have that capability contained within this software would be revolutionary., to say the least.
![image](https://github.com/story-apps/starc/assets/13594657/e07fd077-4f29-4100-ae47-f3f5cd957d17)
